packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -103,14 +113,37 @@ class ImageUpload extends Component<any,any> {
     if (!cloudinaryArgs) throw new Error("Unsupported image upload type")
     // @ts-ignore
     cloudinary.openUploadWidget({
-      cropping: "server",
-      cloud_name: cloudinaryCloudNameSetting.get(),
+      multiple: false,
+      sources: ['local', 'url', 'camera', 'facebook', 'instagram', 'google_drive'],
+      cropping: true,
+      cloudName: cloudinaryCloudNameSetting.get(),
       theme: 'minimal',
-      cropping_validate_dimension: true,
-      cropping_show_dimensions: true,
+      croppingValidateDimensions: true,
+      croppingShowDimensions: true,
+      styles: {
+        palette: {
+            tabIcon: forumThemeExport.palette.primary.main,
+            link: forumThemeExport.palette.primary.main,
+            action: forumThemeExport.palette.primary.main,
+            textDark: "#212121",
+        },
+        fonts: {
+            default: null,
+            "'Merriweather', serif": {
+                url: "https://fonts.googleapis.com/css?family=Merriweather",
+                active: true
+            }
+        }
+      },
       ...cloudinaryArgs
     }, this.setImageInfo);
   }
+  
+  removeImg = () => {
+    this.props.clearField();
+    this.setState({imageId: null});
+  }
```

Comment:
Hmm I wonder if we should actually delete images from Cloudinary if they are removed? Does that matter?